docs/docs/how-to/custom-configuration/typescript.md

```diff
@@ -133,6 +193,66 @@ const getServerData: GetServerData<ServerDataProps> = async props => {
 }
 ```
 
+## `gatsby-config.ts`
+
```

Comment:
The formatting on the code blocks here is off

Reply:
Not sure what you mean here

Reply:
- The first and last line is a blank line
- We don't use semicolons
- Types don't have semicolons in them